### PR TITLE
Update publish docs

### PR DESCRIPTION
**Pull request by kennethlimcp**
- mention about public/private flags
- updated the syntax example

Signed-off-by: Kenneth Lim <kennethlimcp@gmail.com>

### CODE REVIEW

**Comment by kennethlimcp** [comment]
@brycekahle there's some weirdness with the tabs as usual but i hope to set it up properly on my Atom IDE to prevent this the next time

File: README.md [hunk 1]
```diff
@@ -413,7 +414,19 @@ $ particle subscribe eventName deviceName
 $ particle subscribe eventName 0123456789ABCDEFGHI
 ```
 
+### particle publish
+
+	Allows a message to be published via the CLI without using a physical Particle device. This is particularly useful when you are testing your firmware against an actual `published` event.
+
+	There is a `--private` flag that allows you to `publish` events to devices subscribing to events with the `MY_DEVICES` option. You can also explicitly use the `--public` flag though that is the default setting.
```
Comment (brycekahle) [inline README.md]: The `--public` flag does not exist, so I don't think we should mention it.

**Comment by kennethlimcp** [comment]
Oh i thought i had that implemented after a discussion with Mat. Will update the docs.

**Comment by kennethlimcp** [comment]
@brycekahle updated the docs nows

**Comment by brycekahle** [comment]
:+1:

**Pull request merged**